docs/configuring-etcd-options.md

```diff
@@ -0,0 +1,5 @@
+# Configuration Etcd Options
+
+Additional configuration options can be set using the `.spec.etcdOptions` field. The options provided are passed as command line arguments to the etcd container.
+
+Information about the different configuration options is available from the etcd documentation page here: https://etcd.io/docs/v3.5/op-guide/configuration/. Please note that this link is for version 3.5 of etcd and you may need to adjust it for the version of etcd being run.
```

Comment:
Suggest we use permalink

```suggestion
Information about the different configuration options is available from the etcd documentation page here: https://etcd.io/docs/latest/op-guide/configuration. Please note that this link is for version 3.5 of etcd and you may need to adjust it for the version of etcd being run.
```

Reply:
Updated link and added a paragraph about overrides